### ISSUE TITLE
Changes to DSS Introduction content

### ISSUE THREAD

**Issue by hadinon** [ncsco/pinemap-dss-help#93]
Based on the "F pattern" concept we learned from the eye tracking study, move around information within the DSS Introduction tabs as follows:
- Background: flip order of bullets
- About DSS Tools: make categories in 1st bullet into 4 sub-bullets; remove 2nd and 3rd bullets; combine this tab with Background tab ("Tools are broken into 4 categories..." bullet becomes 2nd bullet and "Note: DSS Tools work best with..." bullet becomes last bullet)
- Climate Data: make new category/tab of Emissions (put it right after Climate Data tab), which would contain the information in Climate Data's 3rd bullet; move information about data sources/links (1st two bullets) to DSS development page; re-write this tab so that 1st header is "Future Projections" and 1st bullet = summary of future climates from 20 downscaled climate models & 2nd bullet = available in 4 future time periods & 3rd bullet = from University of Idaho MACA dataset (link to MACA) AND 2nd header is "Historical Data" with 1st bullet = assessment of recent conditions from 1986 to 2005 (maybe add something about how this puts the future projections into context and/or is useful for comparison with future projections) & 2nd bullet = from University of Idaho METDATA dataset (link to METDATA)
- Three-Map Layout: leave as is except maybe remove side maps under historical data?
- Time Series: flip order of bullets; remove information about historical observed bar?

Another idea (probably for the future):
- instead of having an image within each tab/section of the DSS Intro, add a short video highlighting some map features and FAQs/tooltips?

**Comment by daviswx** [comment]
The DSS Introduction page has been updated with the suggested text.  One new tab (Emissions Scenarios) has been added, too, and I moved the RCP graphs into that image space.

I imagine we will want a new image for the Climate Data tab since the bullets repeat the text in the image, but I didn't remember deciding what the new image might be.

Since these updates have pared down the text on each tab, I was thinking that we could also put links to the updated DSS FAQ page below the text at the bottom of each tab.

**Comment by daviswx** [comment]
I have added a new image to the Climate Data tab to replace the one with two screens and text captions about historical and future data.

This one isn't all that informative -- it's pretty much just a composite of three different maps on a computer screen -- but I figure it's a decent stand-in until we can figure something else out.

**Comment by daviswx** [comment]
Suggestion: Go back to the previous image for the Climate Data tab but remove the captions.

Also, explore increasing the font size for the text on the landing page.

**Comment by daviswx** [comment]
A new image is in place on the Climate Data tab.  It's the same as the old one but without the full text/bullet points.

And I have bumped up the font size on the landing page, which should make it a bit more prominent.

**Comment by daviswx** [comment]
These changes got the okay from Ryan today so they will be included in the next push to the public site.

For now, there are no immediate tasks related to the DSS Introduction page content, but once the new DSS FAQ page is ready, I'd like to add links at the bottom of each tab to parts of the FAQ page with more information (e.g., more about the MACA data or emissions scenarios).

**Comment by hadinon** [comment]
Suggestion from the annual meeting last week: 
- On the Intro page (Background tab), instead of saying, "The tools in the PINEMAP DSS use...", re-word as, "You can use the DSS tools..."

**Comment by daviswx** [comment]
I tweaked this wording for the first bullet to say:

_You can use tools in the PINEMAP DSS to explore **climate-based risks and opportunities** for loblolly pine growth_

Since this got rid of the part about productivity models, I moved that bullet to the **About DSS Tools** tab:

_DSS tools are based on **climate data** and **regional ecological and productivity models**_

Let me know if you have any suggested tweaks to this wording.

Other than that, the only other task for the Introduction Page is to add links to FAQ page items at the bottom of each tab.  For example, the **Background** tab might link to the _What is PINEMAP?_ and _What is the PINEMAP DSS?_ questions.

**Comment by hadinon** [comment]
Yep, wording looks good. :+1:  Revisiting the FAQ doc again is on my list for this afternoon.

**Comment by hadinon** [comment]
I have a suggestion for the Emissions Scenarios tab. In particular, I have a suggested tweak to the part that says, "...and radiative forcing -- the greenhouse effect ...". Some people refer to radiative forcing _by trace gases_ as the greenhouse effect (as explained [here](http://www.esrl.noaa.gov/research/themes/forcing/)). However, radiative forcing as a whole also includes more than just trace gases, e.g., changes in the albedo of earths surface and changes in the sun's output. I like how [this](http://news.mit.edu/2010/explained-radforce-0309) article by MIT explains it: "radiative forcing is a direct measure of the amount that the Earth’s energy budget is out of balance". Perhaps we can link to this article or re-word that description slightly?

**Comment by hadinon** [comment]
Any updates on this most recent change? I know things are super busy this week but I think it's an important change for the public facing site. Thoughts?

**Comment by daviswx** [comment]
Sorry about that, this completely slipped under my radar.

I'm fine with rewording this, but I do worry that including a term like "earth's energy budget" could cause additional confusion if users aren't familiar with that either.

What about saying something like this:

"Future projections use two different Representative Concentration Pathways (RCPs), or scenarios that represent potential changes in CO₂ emissions and radiative forcing -- the additional heat energy reaching the earth's lower atmosphere because of the greenhouse effect, fluctuations in the sun's output, and changes in the land surface."

I suppose the downside of this is that it's longer, and as we know from the eye tracking, that means it probably won't be read.

Here's a shorter option:

"Future projections use two different Representative Concentration Pathways (RCPs), or scenarios that represent potential changes in CO₂ emissions and radiative forcing, or the additional heat energy reaching the lower atmosphere."

**Comment by hadinon** [comment]
We're getting close! Even though this is not currently the case, the imbalance could potentially be negative (and thus, have a cooling effect). Plus, I just noticed that we only mention CO2 as greenhouse gas emissions that are represented by RCPs but other greenhouse gases are included, too. So, how about this?

"Future projections use two different Representative Concentration Pathways (RCPs), or scenarios that represent potential changes in greenhouse gas emissions (e.g., CO₂) and radiative forcing (e.g., energy flowing in minus energy flowing out of the atmosphere)."

**Comment by daviswx** [comment]
Could we make the radiative forcing explanation less math-y?  What about this:

"... and radiative forcing (the difference between the incoming and outgoing energy in the atmosphere)."

I also wondered whether we should include a clause like "... from both manmade and natural sources" at the end of this.

**Comment by hadinon** [comment]
Yeah, that sounds good to me. Thanks!

**Comment by daviswx** [comment]
Okay, I have made that change on the development site.  I haven't copied it over to the public site yet since I'm currently going through the code and cleaning up some things to help make the Climate Voyager setup a bit easier.  Once that's done, I will commit this change and pull it down on the public site.

**Comment by hadinon** [comment]
Sounds good. I might suggest one more minor wording tweak to the RCP4.5 explanation. I don't mean to be nitpicky. It's been REALLY difficult to balance brevity, clarity, and precision in these explanations. 

In the explanation for RCP4.5, we say, "...emissions and radiative forcing level off or decline by mid-century", which is technically true. However, my hesitation is that it will be implied that radiative forcing declines, which is not true. I hope that makes sense. I might suggest: "...emissions decline and radiative forcing levels off by end of century". Thoughts?

**Comment by daviswx** [comment]
That is updated on the dev site.  Nitpicky things like that are definitely good for these explanations so they're as clear and accurate as possible.

I'm still cleaning up a few files on the dev site to clean up some clutter, but once that's done, I will push the recent changes (minus model error) to the public site.

**Comment by daviswx** [comment]
The public version of the DSS has been updated to reflect the new wording on the Emissions Scenario tab.

**Comment by daviswx** [comment]
Another task for the DSS Introduction page: At some point, the images for the **About DSS Tools**, **Three-Map Layout**, and **Time Series** tabs need to be updated to reflect some recent-ish changes to the DSS look, like the removal of the search box and the range of historical variability.

**Comment by daviswx** [comment]
Now that we have made more progress on the FAQ pages, we may want to revisit an idea that we had for the Introduction page.  At one point, we talked about linking to specific FAQs at the bottom of each tab on the Introduction page, e.g., the "Emissions Scenarios" tab could link to that FAQ.

If we still want to do this, here are some thoughts about which FAQs we might link to:
- **Background tab**: _What is the PINEMAP DSS/Climate Voyager and what can it be used for?_
- **About DSS Tools tab** (PINEMAP DSS only): _How can I find tools?_ (or no link at all for this one; I sort of question whether we need this tab at all)
- **Climate Data tab**: _What is being shown on the historical data display and what is the data source?_ and _What is the data source for the future climate projections?_
- **Emissions Scenarios tab**: _What are the two emissions scenarios used?_
- **Three-Map Layout tab**: _What are the multi-model mean and the lowest and highest likely change or outcome?_
- **Time Series tab**: _What is being shown in the time series plot and what do the colored bars represent?_

**Comment by hadinon** [comment]
Sounds good about the updated images for those three tabs you mentioned.

Yes, let's add these links! 
- **Background tab**, I wonder if we should also link to the _What is PINEMAP?_ one (PINEMAP DSS only)?
- **About DSS Tools tab**: that seems like a good one to link. You made a comment about questioning whether we need this tab at all. Did you mean whether we need to remove the "About DSS Tools" tab or are you questioning whether we even need to add a link to this tab?
- **Time Series tab**: add this one? _How can I view the time series plot?_

I agree with your other suggested links.

**Comment by daviswx** [comment]
Sounds fine about adding those other links.

Re: the "About DSS Tools" tab, I'm wondering whether we need that actual tab.  We don't have a similar tab on Climate Voyager, and we just put the information about browser versions onto the "Background" tab.  We could do the same thing on the DSS; the menu explanations don't seem super important in my mind, and we could either absorb the information about climate data/productivity models onto the "Background" tab or into one of the FAQs (maybe the _What is the PINEMAP DSS and what can it be used for?_ one).

**Comment by hadinon** [comment]
OK yeah, I'm OK with removing that tab. I agree about moving the browser information onto the "Background" tab and removing the menu explanations. Also, we sort of already say the 2nd bullet from "About DSS Tools" within the 1st bullet of the "Background" tab so we should be covered there, too.

**Comment by daviswx** [comment]
The Introduction page has been updated to add those links at the bottom of each tab and remove the "About DSS Tools" tab.  Also, I have removed the MACA FAQ page and changed the link to the current FAQ page.

I have tested the updates on both the DSS and Climate Voyager development sites and everything seems to be working correctly, but as always, please re-open this issue if you notice any problems.

**Comment by hadinon** [comment]
All looks good! Two minor questions (that don't warrant re-opening IMO):

1) Should the links at the bottom of each tab open in a new browser tab?
2) It was mentioned about that at some point, the images for the Three-Map Layout and Time Series tabs on the PINEMAP DSS need to be updated to reflect some recent-ish changes to the DSS look, like the removal of the search box and the range of historical variability. Has this been done? Maybe I should clear my cache, haha!!
